examples/http_server/module/src/lib.rs

```diff
@@ -76,15 +78,26 @@ impl CommandHandler for StaticHttpHandler {
             "/" => http::response::Builder::new()
                 .status(http::StatusCode::OK)
                 .header(http::header::CONTENT_TYPE, "text/html; charset=UTF-8")
-                .body(include_bytes!("../static/index.html").to_vec()),
+                .body(include_bytes!("../static/index.html").to_vec())
+                .context("Could not build response"),
+            "/test_google" => {
+                // create a public HTTP client pseudo-node
+                let http_client = oak::http::client::init("")
+                    .context("Could not create HttpClient pseudo-node")?;
+                let request = http::Request::builder()
+                    .method(http::Method::GET)
+                    .uri("http://www.google.com")
+                    .body(vec![])
+                    .context("Could not build request")?;
+                http_client
+                    .send_request(request, &Label::public_untrusted())
+                    .context("Could not get response")
```

Comment:
FYI Creating a new node from another node is not very realistic (most of the examples now create all the nodes in the main node). Also it seems this example is not really exercising TLS label, but given how it is written already it could easily do that too, right? For instance by handling data labelled with https:google.com TLS label.